### PR TITLE
feat: render become delegator quest

### PR DESCRIPTION
**Pull request by juliano-quatrin-nunes**
Preview: 
image

### CODE REVIEW

File: apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/become_delegate.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module ActionTracking
+  module Strategies
+    class BecomeDelegate < Base
```
Comment (ribeirojose) [inline apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/become_delegate.rb:3]: this should be `BecomeDelegator` right?

File: apps/govquests-frontend/src/domains/questing/components/QuestCard.tsx [hunk 2]
```diff
@@ -4,17 +4,19 @@ import type React from "react";
...
   quest: Quests[number];
 }
 
 const QuestCard: React.FC<QuestProps> = ({ quest }) => {
+  const questLink = `/quests/${titleToSlug(quest.displayData.title)}`;
+
   return (
     <Link
-      href={`/quests/${quest.id}`}
+      href={questLink}
```
Comment (ribeirojose) [inline apps/govquests-frontend/src/domains/questing/components/QuestCard.tsx:19]: why wouldn't there be a `quest.slug`? 🤔

File: apps/govquests-frontend/package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev --turbopack",
```
Comment (ribeirojose) [inline apps/govquests-frontend/package.json:6]: 👍

**Review by ribeirojose — APPROVED** [review]
in essence lgtm, but think I'd have taken a different route of adding the slug directly to the quest entity. Have you considered that?

File: apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/become_delegate.rb (hunk 1 above)
Comment (juliano-quatrin-nunes) [reply apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/become_delegate.rb:3]: Right! Fixed on #81

**Review by ribeirojose — APPROVED** [review]
🥳 good stuff!

**Pull request merged**